test/unit/active_model/serializer/has_many_polymorphic_test.rb

```diff
@@ -2,7 +2,7 @@
 
 module ActiveModel
   class Serializer
-    class HasManyPolymorphicTest < ActiveModel::TestCase
+    class HasManyPolymorphicTest < Minitest::Test
```

Comment:
what's the reason for this change in this PR? Is it required?

Reply:
I guess I misread this as removing ActiveSupport::TestCase (and not ActiveModel::TestCase)

I recall in 0.10x we went with ActiveSupport::TestCase over Minitest::Test since it provided better compatibility across minitest versions

Reply:
Personally have never worked with minitest and i notice most of these tests were written back in 2013/14. I made the switch simply by referencing what other model test files were referencing. I thought it made more sense for them to me the same across the board.